lib/utils/message-truncator.js

```
@@ -0,0 +1,153 @@
+
+    const parts = [];
+    let remaining = message;
+    let partNumber = 1;
+    const totalParts = Math.ceil(message.length / maxPartLength);
+
+    while (remaining.length > 0) {
+      let currentPart = remaining.substring(0, maxPartLength);
+      
+      // Try to break at a natural boundary
+      if (remaining.length > maxPartLength) {
+        currentPart = this.truncateAtNaturalBoundary(remaining, maxPartLength);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)